src/component/InvoiceContainer/index.jsx

```diff
@@ -0,0 +1,33 @@
+import { useEffect, useState } from "react"
+import Invoice from "../Invoice"
+import Footer from "../Footer"
+import Header from "../Header"
+
+
+function InvoiceContainer() {
+    const [invoices, setInvoices] = useState([])
+    useEffect(() => {
+        // Instructions here on what you need to happen on component mount
+        fetch('https://invoicing-api.dev.io-academy.uk/invoices')
+            .then(res => res.json())
+            .then((invoiceData) => {
+                setInvoices(invoiceData.data)
+
+            })
+    }, [])
+
+    return (
+        <>
+            <Header />
+
+
+            {invoices.map(invoice => <Invoice invoiceTotal={invoice.invoice_total} invoiceStatus={invoice.status_name} clientName={invoice.name} invoiceId={invoice.invoice_id} dueDate={invoice.due} />)}
```

Comment:
You need to give the Invoice a key prop - you'll be getting a warning in the console about this.

Also, this line is getting pretty long. I suggest putting each prop onto it's own line. 

<img width="751" alt="Screenshot 2024-07-02 at 15 28 30" src="https://github.com/iO-Academy/2024-may-invoicing-sloth-rockets/assets/19924378/8c4fa54c-0b47-40c1-aae6-c8d88a1a13fa">